src/main/groovy/org/mockito/release/notes/vcs/DefaultCommitApprover.java

```diff
@@ -0,0 +1,31 @@
+package org.mockito.release.notes.vcs;
+
+import org.mockito.release.notes.model.Commit;
+import org.mockito.release.notes.util.Predicate;
+
+import java.util.Collection;
+import java.util.LinkedList;
+
+public class DefaultCommitApprover implements Predicate<Commit> {
```

Comment:
Design suggestion: it might be simpler to just have a single implementation of the Predicate, and make the impl work with a list of Strings. Something like that: ```new DefaultCommitApprover("[ci skip]", "[ci skip-release]")```.

The simpler code, the easier testing and easier to maintain :)